### DISCUSSION TITLE
!pip install -q git+https://github.com/pyannote/pyannote-audio is not working in google colab

### DISCUSSION THREAD

**Discussion by kaliNight** [pyannote/pyannote-audio#1528] [Q&A]
following by the below erorr
"WARNING: Ignoring invalid distribution -orch (/usr/local/lib/python3.10/dist-packages)
WARNING: Ignoring invalid distribution -orch (/usr/local/lib/python3.10/dist-packages)
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
openai-whisper 20230918 requires triton==2.0.0, but you have triton 2.1.0 which is incompatible."

I tried to install triton 2.0.0 but it still using trition 2.1.0

**Comment by hbredin** [comment]
I think the problem is not related to `pyannote.audio` as neither `openai-whisper` nor `triton` is in `pyannote` list of dependencies.

**Comment by dgoryeo** [comment]
@kaliNight , I think one workaround is to downgrade your triton as well as torch, torchaudio and torchvision.
